src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/AZC0018Tests.cs

```diff
@@ -21,8 +21,6 @@ public async Task AZC0018NotProducedForCorrectReturnType()
 
 namespace RandomNamespace
 {
-public class AssetConversion {}
```

Comment:
Curious: why is this being removed?

Reply:
I'm not removing. I think you are not seeing the right iteration.